.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
It's actually already added, just didn't show up in the diff 😺

Reply:
hmm, wdym by didn't show up in the diff?

Reply:
https://github.com/hansel-app/hansel/blob/weim/frontend-setup/frontend/.gitignore

Reply:
oh, but I think we may need another .gitignore in the root directory, esp if you open the repo from the root directory rather than the separate `frontend` or `backend` directory

Reply:
Ohh okay! Got it